shared/shared/postgresql_backend/schema.sql

```diff
@@ -33,6 +33,7 @@ CREATE TABLE IF NOT EXISTS events (
     data JSONB
 );
 CREATE INDEX IF NOT EXISTS events_fqid_idx ON events (fqid);
+-- TODO: create index for data->>meeting_id for collectionfieldlocks
```

Comment:
Still left

Reply:
Yes, I would like to test that first with performance tests before integrating an index which may be not bring any benefit